### ISSUE TITLE
Seedphrase recovers different accounts

### ISSUE THREAD

**Issue by osmaczko** [status-im/status-desktop#5253]
Accounts recovered from the same seedphrase (copy-pasted) are different!

![image](https://user-images.githubusercontent.com/33099791/160653935-f0c5ad4b-486e-401c-a4b5-c9b132a6cefb.png)
![image](https://user-images.githubusercontent.com/33099791/160653987-7af03b1c-269d-495c-94e1-5bf24129bc73.png)
![image](https://user-images.githubusercontent.com/33099791/160654040-13573ef6-e3f0-4bf3-945c-5c3ce34ba1b6.png)

**Comment by iurimatias** [comment]
![Screen Shot 2022-03-29 at 12 04 46 PM](https://user-images.githubusercontent.com/176720/160656625-262b0a5e-0fee-4c8e-9cab-7bdb69762ae8.png)
![Screen Shot 2022-03-29 at 12 04 27 PM](https://user-images.githubusercontent.com/176720/160656630-539a521f-f92c-434e-814d-eabfc9c19fe2.png)
![Screen Shot 2022-03-29 at 12 04 02 PM](https://user-images.githubusercontent.com/176720/160656631-1a17f025-0aef-40e8-b141-e64143d66fb4.png)
![Screen Shot 2022-03-29 at 12 03 25 PM](https://user-images.githubusercontent.com/176720/160656633-6307d46c-9e63-46ce-9fd6-7d33e2c03b29.png)

**Comment by osmaczko** [comment]
```
$ git bisect bad                                                                                          
dcab50fe0944f65a1fafef3fdb9e7abe66277b6c is the first bad commit
commit dcab50fe0944f65a1fafef3fdb9e7abe66277b6c                                                           
Author: Alexandra Betouni <alexandra.betouni@protonmail.com>
Date:   Tue Mar 8 00:59:38 2022 +0200                                                                     

    feat(onboarding): ading create new keys screens
     
    Closes #4956
```
